app/components/App/App.css

```diff
@@ -0,0 +1,15 @@
+header {
+  background-color: orange;
```

Comment:
oh man,  I love orange.  Go Tigers.  (gators suck)

Reply:
You can't pick and choose teams. COLORS DONT FADE